doc/examples/example5.md

```diff
@@ -0,0 +1,397 @@
+# Asymmetric Crypto API {#example5}
+
+# Common Interface
+
+## Encryption Interface
+
+All contexts that support en- / decryption are implementing an `encrypt()` or `decrypt()` method resprectively. The classes `mococrw::EncryptionCtx` and `mococrw::DecryptionCtx` provide a pure virtual interface to these methods:
+<!-- TODO for when refactoring Examples: Generate such code snippets from code-->
+\code{.cpp}
```

Comment:
As a general question about this documentation: These are markdown files, but we're using doxygen tags `\code{.cpp}` in them. Is there a rendered version of these available somewhere? Otherwise I would probably to use markdown code blocks instead so these examples are at least readable on GitHub.

Reply:
I was already talking to @vkrikun about this. The contents of these md files are embedded into the generated doxygen documentation. We agreed on looking if there are better solutions to this as part of a separate task (as far as i remember there is already such a ticket somewhere in the backlog)